doc/docs/reference/inter-workload-dependencies.md

```diff
@@ -0,0 +1,139 @@
+
+# Inter-workload dependencies
+
+Ankaios allows a user to configure dependencies between workloads.
+
+Ankaios supports two types of inter-workload dependencies:
+
+- explicit inter-workload dependencies
+- implicit inter-workload dependencies
+
+The [explicit inter-workload dependencies](#explicit-inter-workload-dependencies) are configured by the user inside a workload's configuration and considered by Ankaios when starting a workload. Ankaios starts workloads having dependencies only when all of its dependencies are met. In this way, the user can define a specific sequence in which the workloads are started.
+
+The [implicit inter-workload dependencies](#implicit-inter-workload-dependencies) are defined by Ankaios internally and considered when a dependency itself is deleted.
+
+## Explicit inter-workload dependencies
+
+Ankaios supports the following dependency types:
+
+| Dependency type |  AddCondition         | meaning                                       |
+| --------------- | --------------------- | --------------------------------------------- |
+| running         | ADD_COND_RUNNING      | the dependency must be operational            |
+| succeeded       | ADD_COND_SUCCEEDED    | the dependency must be exited successfully    |
+| failed          | ADD_COND_FAILED       | the dependency must be failed                 |
+
+A user defines one or multiple dependencies for an workload by configuring the `AddCondition` for each dependency within the field `dependencies`:
+
+```yaml
+logger:
```

Comment:
I would use the format of an Anakios manifest here, like

```yaml
apiVersion: v1
workloads:
  logger:
    ...
```

to get a better impression of the context.

Reply:
I have updated it.